README.md

```diff
@@ -723,6 +722,66 @@ This is an incredibly powerful tool that comes baked-in with elixir. It means th
 other developers who are joining the project can be brought up to speed incredibly
 quickly!
 
+## Testing
+
+When you generate a project with Elixir it automatically gives you a number of
+files and directories. One of these directories is called `test` and it holds two
+files like should have names like:
+- `[project_name]_test.exs`
+- `test_helper.exs`
+
+Our first file was called `animals_test.exs` and it contained some boilerplate that
+looks like:
+
+```elixir
+defmodule AnimalsTest do
+  use ExUnit.Case
+  doctest Animals
+
+  test "the truth" do
+    assert 1 + 1 == 2
+  end
+end
+```
+**NOTE: It automatically includes a line called `doctest Animals`. What this means
```

Comment:
Does the _whole_ **NOTE: ...**" need to be bold 
or _could_ we make **Note** in **bold** and the rest of the text normal? 🤔